android/src/main/java/com/tailscale/ipn/ui/model/TailCfg.kt

```diff
@@ -97,7 +99,7 @@ class Tailcfg {
       get() = Name.endsWith(".mullvad.ts.net.")
 
     val displayName: String
-      get() = ComputedName ?: ""
+      get() = ComputedName
```

Comment:
this field might be omitted sometimes right? https://github.com/tailscale/tailscale/blob/main/tailcfg/tailcfg.go#L357

Reply:
Good catch.  That was an auto correction.  It's not optional in the schema and should be.  I will audit the other fields too.

Reply:
Fixed.  Switched to node.displayName throughout, which uses ComputedName ?: Name...